test/unit/org/apache/cassandra/tools/nodetool/CompactionStatsTest.java

```diff
@@ -130,6 +130,14 @@ compactionId, OperationType.COMPACTION, CQLTester.KEYSPACE, currentTable(), byte
                                                     CompactionInfo.Unit.BYTES, (double) bytesCompacted / bytesTotal * 100);
         Assertions.assertThat(stdout).containsPattern(expectedStatsPattern);
 
+        Assertions.assertThat(stdout).containsPattern("[0-9]* concurrent compactors, [0-9]* pending tasks");
```

Comment:
I think to be on the safe side, you can add a test case that uses the invokeNodetool method for full-link testing.
Though when using invokeNodetool from a client side , It is impossible to specifically determine the value of the test, but we can test and output some expected results, such as whether the output information has the print out field we need, and when there is no data, and when we write a certain amount of data then do a major, These print out data should met our expectations。
@driftx WDYT?

Reply:
I think it is just fine as is.